docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -7,16 +7,15 @@ weight: 10
 
 # Set up Go profiling in pull mode
 
-In pull mode, the Grafana Agent periodically retrieves profiles from Golang applications, specifically targeting the
+In pull mode, the collector, whether Grafana Alloy or Grafana Agent (legacy), periodically retrieves profiles from Golang applications, specifically targeting the
 `/debug/pprof/*` endpoints.
 
-
 To set up Golang profiling in pull mode, you need to:
 
 1. Expose pprof endpoints
-2. Install Grafana Agent
-3. Prepare Grafana Agent configuration file
-4. Start Grafana Agent
+2. Install a collector, either Grafana Alloy (preferred) or Grafana Agent
```

Comment:
Same question nere... this spot missed `(legacy)`, but I'm thinking both could be dropped once it's stated up at the top of the topic.